encoding/v2/kzg/prover/backend/icicle/multiframe_proof.go

```diff
@@ -146,10 +160,14 @@ func (p *KzgMultiProofBackend) ComputeMultiFrameProofV2(ctx context.Context, pol
 
 		msmDone = time.Now()
 
-		// Compute the first ecntt, and set new batch size for ntt
-		p.NttCfg.BatchSize = int32(1)
+		// Create NTT config for ECNTT operations
+		nttCfg := ntt.GetDefaultNttConfig()
+		nttCfg.IsAsync = true
+		nttCfg.Ordering = core.KNN
+		nttCfg.StreamHandle = stream
+		nttCfg.BatchSize = int32(1)
```

Comment:
```suggestion
		nttCfg.StreamHandle = stream
```

Reply:
nit: dont need to set default values.

Reply:
https://github.com/Layr-Labs/eigenda/pull/2269/commits/7d82771087a33cf42b38bafc9f78f2dff1201201